src/CommunityToolkit.Aspire.Hosting.Deno/DenoAppHostingExtensions.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting;
+using Aspire.Hosting.Lifecycle;
+using Aspire.Hosting.Utils;
+using Microsoft.Extensions.Hosting;
+
+namespace CommunityToolkit.Aspire.Hosting.Deno;
+public static class DenoAppHostingExtensions
```

Comment:
```suggestion
/// <summary>
/// Extension methods to support adding Deno to the <see cref="IDistributedApplicationBuilder"/>.
/// </summary>
public static class DenoAppHostingExtensions
```

This will avoid a problem with #151